waspc/tools/install_packages_to_data_dir.bat

```diff
@@ -0,0 +1,49 @@
+@echo off
+setlocal enableDelayedExpansion 
+
+REM Window BAT script to compile the waspc/packages/* packages locally and in CI.
+REM It will then move it into the Cabal data dir (and thus, the installer archive in CI releases).
+REM This script derived from tools/install_packages_to_data_dir.sh using Copilot (6 Nov. 2025).
+REM Reviewed, revised, and commented by @nodtem66 (6 Nov. 2025)
+
+REM Gets the directory of where this script lives.
+set dir=%~dp0
+REM Remove trailing backslash
+if "%dir:~-1%"=="\" set "dir=%dir:~0,-1%"
+
+REM set alias cmd for cp
+DOSKEY cp=COPY $* 
+
+REM Loop through each package directory
+REM For /d Loop to iterate over directories
+for /d %%G in ("%dir%\..\packages\*") do (
+    set package_dir=%%G
+    set package=%%~nxG
+
+    echo Installing !package! ^(!package_dir!^)
+    cd /d "!package_dir!"
+    
+    REM We're only installing the dependencies here to verify that the build
+    REM works, that's why the node_modules folder is removed immediately after.
+    REM The real dependency installation happens in Haskell.
```

Comment:
Maybe let's try to replicate the original script as close as we can and put this above the `echo Installing ...`, same like in the original script.